src/settings/index.js

```diff
@@ -18,13 +24,29 @@ export default () => {
     region: getEnvParam('FUNCTION_REGION'),
   };
   const { decryptSecret } = decrypter(baseConfig);
-
-  const getConfig = async () => {
+  const getConfig = async (harvestAccount = 'mavericks') => {
     const secretConfigString = await decryptSecret();
     const secretConfig = JSON.parse(secretConfigString);
+    const envHarvest = process.env.HARVEST ? process.env.HARVEST : '';
```

Comment:
Is this documented somewhere? Or set in deployment to default?

Reply:
This is actually unused piece of code. Remnants of first draft implementation for local CLI setup to use env variable for selecting account. It is implemented to given CLI command itself.
Removed this settings implementations since it is unnecessary.